src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/JJDisputeController.java

```diff
@@ -95,11 +96,93 @@ public ResponseEntity<JJDispute> updateJJDispute(
 			Principal principal,
 			@RequestBody JJDispute jjDispute) {
 		logger.debug("PUT /dispute/{ticketNumber}/{checkVTCAssigned} called");
-		if (checkVTCAssigned == true) {
-			if (!jjDisputeService.assignJJDisputeToVtc(ticketNumber, principal)) {
-				return new ResponseEntity<>(null, HttpStatus.CONFLICT);
-			}	
+		
+		if (checkVTCAssigned && !jjDisputeService.assignJJDisputeToVtc(ticketNumber, principal)) {
+			return new ResponseEntity<>(null, HttpStatus.CONFLICT);
 		}
+		
 		return new ResponseEntity<JJDispute>(jjDisputeService.updateJJDispute(ticketNumber, jjDispute, principal), HttpStatus.OK);
 	}
+	
+	/**
+	 * PUT endpoint that updates each JJ Dispute based on the passed in IDs (ticket number) to assign them to a specific JJ or unassign them if JJ not specified.
+	 *
+	 * @param ID(s) (ticket number) of JJ Dispute(s) to be updated/assigned
+	 * @param IDIR username of the JJ that JJ Dispute(s) will be assigned to, if specified. Otherwise JJ Disputes will be unassigned
+	 */
+	@Operation(summary = "Updates each JJ Dispute based on the passed in IDs (ticket number) to assign them to a specific JJ or unassign them if JJ not specified.")
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "Ok"),
```

Comment:
Can we add `content` to these @ApiResponse?  If there truely is not content (400, etc), then you would use `content = @Content`.  However, on Bad Request, what is the return type. How is the callers supposed to decode and log issues?